app/services/media_storage/azure_adapter.rb

```diff
@@ -42,7 +43,7 @@ def get_path(path, opts={})
           expiry: get_expiry_time(opts[:get_expires] || get_expiration)
         ).to_s
       else
-        "https://#{path}"
+        StoredPath.media_url(url, path)
```

Comment:
Rather than storing/hardcoding the url in a env var, we could use the Blob Client's method `@client.generate_uri`, which will return the microsoft domain based off of the storage account name it was initialized with as a URI, e.g. `#<URI::HTTPS https://panoptes.blob.core.windows.net/>
`. That way, if we change the `AZURE_STORAGE_ACCOUNT` env var, we wont also have to change the `STORAGE_URL` env var. In this case, we wouldn't even need THE `STORAGE_URL` env var.

Reply:
We want to be able to specify this to be different from the blob, i.e. our CDN URL for production deployment. However we want to be able to test out canary / staging etc via the direct path to the storage account container data path. 

For our production deployment we can't break existing URLs so we plan to use `panoptes-uploads.zooniverse.org` as a singular CDN target (exactly how the production & staging systems currently work). We wont' expose the windows blob URLs to our consuming clients (only in canary testing etc)